browser/src/slideshow/LayerRenderer.ts

```diff
@@ -129,16 +127,15 @@ class LayerRendererGl implements LayerRenderer {
 
 		let vertices = LayerRendererGl.DefaultVertices;
 		if (bounds) {
+			vertices = [];
 			// convert [0,1] => [-1,1]
-			for (let i = 0; i < bounds.length; ++i) bounds[i] = 2 * bounds[i] - 1;
-
-			// flip y coordinates
-			const l = bounds[0]; // left bound
-			const t = -bounds[1]; // top bound
-			const r = bounds[2]; // right bound
-			const b = -bounds[3]; // bottom bound
-
-			vertices = [l, b, r, b, l, t, l, t, r, b, r, t];
+			for (let i = 0; i < bounds.length; ++i) {
+				const x = 2 * bounds[i].x - 1;
+				vertices.push(x);
+				// flip y coordinates
+				const y = -(2 * bounds[i].y - 1);
+				vertices.push(y);
+			}
```

Comment:
Because here I can just convert to [-1,1] the 4 vertices of the bounding box and it's done; no need to add the same vertex more than once.
Is there a reason you preferred to use TRIANGLES instead of TRIANGLE_STRIP as you did in transitions and slide renderer ?